### ISSUE TITLE
Bug in Project Configuration

### ISSUE THREAD

**Issue by asclines** [ASAAR/SE2-KaggleComp#147]
After commit #9a939967e8e3b508fe39b711e7dff08280119e8f was created, **all** tests for the project configuration tool broke. 
This is the error:
```
ConfigurationXMLError: Required XML tag [ID_label] not found in valid.xml
```
Tests need to be fixed.

**Comment by ZakeryFyke** [comment]
-groan-

**Comment by asclines** [comment]
Closed in #148
